src/desktop/space/window.rs

```diff
@@ -1,21 +1,19 @@
 use crate::{
-    backend::renderer::{ImportAll, Renderer},
-    desktop::{
-        space::Space,
-        window::{draw_window, Window},
-    },
+    backend::renderer::{ImportAll, Renderer, Texture},
+    desktop::{window::Window, PopupManager},
     utils::{Logical, Physical, Point, Rectangle, Scale},
-    wayland::output::Output,
 };
 use std::{
-    any::TypeId,
     cell::{RefCell, RefMut},
     collections::HashMap,
 };
 
+use super::SpaceElement;
+
 #[derive(Default)]
 pub struct WindowState {
     pub location: Point<i32, Logical>,
+    // TODO: How to handle that?
     pub drawn: bool,
```

Comment:
We don't have to handle this anymore (just drop it), it was only used for some ill-guided send_frame *optimization*, that we have removed already.

Reply:
:+1: done